tests/rsa_tests.rs

```diff
@@ -163,26 +185,27 @@ fn test_signature_rsa_pkcs1_verify() {
             // Sanity check that we correctly DER-encoded the originally-
             // provided separate (n, e) components. When we add test vectors
             // for improperly-encoded signatures, we'll have to revisit this.
-            assert!(untrusted::Input::from(&public_key)
-                .read_all(error::Unspecified, |input| der::nested(
-                    input,
-                    der::Tag::Sequence,
-                    error::Unspecified,
-                    |input| {
+            let key_bits = untrusted::Input::from(&public_key)
+                .read_all(error::Unspecified, |input| {
+                    der::nested(input, der::Tag::Sequence, error::Unspecified, |input| {
+                        let n_len = der::positive_integer(input)?
+                            .big_endian_without_leading_zero()
+                            .len();
                         let _ = der::positive_integer(input)?;
-                        let _ = der::positive_integer(input)?;
-                        Ok(())
-                    }
-                ))
-                .is_ok());
+                        Ok(n_len * 8)
```

Comment:
Instead of multiplying out `n_len * 8`, we need to use the exact bit length as returned by `Modulus::from_be_bytes_with_bit_length`, don't we?